content/influxdb/v2.0/tools/flux-vim-lsp.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Use the Flux LSP with Vim
+description: >
+  Use the Flux LSP with Vim
+menu:
+  influxdb_2_0:
+    parent: Tools & integrations
+weight: 102
+---
+
+
+Use the Flux LSP (an implentation of the [Language Server Protocol](https://microsoft.github.io/language-server-protocol/) for Flux)
+with Vim to add auto-completion, syntax checking, and other language-specific features to your editor.
+
+## Requirements
+
+- Vim 8+
+- [npm](https://www.npmjs.com/get-npm)
+
+## Installation
+
+There are many ways to install and manage Vim plugins.
+We recommend the following two methods:
+
+- [Install with vim-lsp](#install-with-vim-lsp)
+- [Install with vim-coc](#install-with-vim-coc)
+
+Both methods require you to add the following to your `.vimrc` so that Vim can recognize the `.flux` file type:
+
+```
+" Flux file type
+au BufRead,BufNewFile *.flux		set filetype=flux
+```
+
+### Install with vim-lsp
+
+1. **Install `flux-lsp-cli` with npm**
+
+    ```
+    npm i -g @influxdata/flux-lsp-cli
+    ```
+
+2. **Install [vim-lsp](https://github.com/prabirshrestha/vim-lsp)**
+
+    If it doesn't exist yet, create a directory called `pack/<user>/start/` in your `.vim/` and clone `vim-lsp` into it:
+
+    ```sh
+    cd ~
+    mkdir -p .vim/pack/<user>/start/
+    cd .vim/pack/<user>/start/
+    git clone https://github.com/prabirshrestha/vim-lsp
+    ```
```

Comment:
- Is `<user>` their system user or does Vim have a concept of users?
- Should `.vim/` be `~/.vim/`?

Reply:
It can be an string, really. I think the standard is to use `$USER`. Might go ahead and recommend that here.

Reply:
Yep. `~/.vim/`.